roles/kubespray_defaults/defaults/main/main.yml

```diff
@@ -632,6 +632,8 @@ ssl_ca_dirs: |-
   {% endif -%}
   ]
 
+# used for delegating tasks on a working control plane node
+first_kube_control_plane: "{{ groups['kube_control_plane'] | first }}"
```

Comment:
This is a nice touch (to put it in default, since it allows people to override it, and potentially explicitly designate a control plane to use as the first. (I'm not completely sure doing that would work, because I think that variable is used for the kubeam upgrade command and maybe we actually rely on the inventory ordering 🤔 . But one could always try ^